docs/app/pages/components/sections/panels/item-display-panel/item-display-panel.component.ts

```diff
@@ -0,0 +1,160 @@
+import { ColorService } from './../../../../../../../src/services/color/color.service';
+import { Component } from '@angular/core';
+import { DocumentationSectionComponent } from '../../../../../decorators/documentation-section-component';
+import { BaseDocumentationSection } from '../../../../../components/base-documentation-section/base-documentation-section';
+import 'chance';
+import { IPlunk } from '../../../../../interfaces/IPlunk';
+import { IPlunkProvider } from '../../../../../interfaces/IPlunkProvider';
+import { ItemDisplayPanelComponent } from '../../../../../../../src/index';
+
+@Component({
+    selector: 'uxd-item-display-panel-component',
+    templateUrl: './item-display-panel.component.html'
+})
+@DocumentationSectionComponent('ComponentsItemDisplayPanelComponent')
+export class ComponentsItemDisplayPanelComponent implements IPlunkProvider {
+
+    visible = false;
+    title = '';
+    top = 0;
+    main = '';
+    selected = 0;
+    previousEnabled = true;
+    nextEnabled = true;
+    
+    sparkBarColor: string;
+    sparkTrackColor: string;
+
+    constructor(colorService: ColorService) {
+        this.sparkTrackColor = colorService.getColor('accent').setAlpha(0.2).toRgba();
+        this.sparkBarColor = colorService.getColor('accent').toHex();
+    }
+
+    htmlCode = require('./snippets/app.html');
```

Comment:
Move these above the constructor as we generally have variable declarations above constructor and methods below, make everything easy to find.